### DISCUSSION TITLE
ES6 Supported?

### DISCUSSION THREAD

**Discussion by Zuzuesque** [vuepress/core#401] [Q&A] [answered]
I recently tried to add vuepress@next to a new typescript project which set the type field in package.json to "module", this is giving me trouble with vuepress however. 

I always get the error: 
``` 
BREAKING CHANGE: The request '@vuepress/client/lib/components/Vuepress' failed to resolve only because 
it was resolved as fully specified (probably because the origin is strict EcmaScript Module, e. g. a module 
with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.

webpack compiled with 1 error
```

I have to admit I haven't worked a lot with either vuepress or webpack in the past and I am at a loss how to resolve the issue. Is it possible to run vuepress in a es6 project? _Can_ I add the extension to the request or specify a package file just for the docs?

**Comment by Mister-Hope** [comment, marked as answer]
Please describe with more detail, as I am not following you. Or at least, provide a minial reproduction repo link.
